content/installation/upgrading-manager.md

```diff
@@ -2,96 +2,113 @@
-  1. Install the Cloudify Manager on the host.
-  1. Restore snapshot of the Cloudify Manager to the host.
-
-* Migration -
+{{% gsWarning title="Downtime" %}}
+Make sure that no users are connected to any manager services during the upgrade or migration.
+{{% /gsWarning %}}
 
-  1. Save snapshot of the Cloudify Manager.
-  1. Install the Cloudify Manager on the host.
-  1. Restore snapshot of the Cloudify Manager to the host.
-  1. (Optional) Uninstall (teardown in Cloudify 4.2 and below) the Cloudify Manager from the host.
-  1. Migrate agents from the old Cloudify Manager.
+The upgrade process includes:
 
 * In-place upgrade -
 
-  1. Save snapshot of Cloudify Manager.
-  1. Uninstall (teardown in Cloudify 4.2 and below) the Cloudify Manager from the host.
-  1. Install the Cloudify Manager latest version on the host.
-  1. Restore snapshot of the Cloudify Manager to the host.
+  1. [Backup a snapshot]({{< relref "installation/upgrading-manager.md#backup-a-snapshot" >}}) of the Cloudify Manager.
+  1. [Backup the agents certificates]({{< relref "installation/upgrading-manager.md#backup-agent-certificates" >}}) from the Cloudify Manager.
+  1. [Uninstall the Cloudify Manager]({{< relref "installation/upgrading-manager.md#uninstall-cloudify-manager" >}}) from the original host. In Cloudify 4.2 and below, this is called teardown.
+  1. [Install the Cloudify Manager]({{< relref "installation/upgrading-manager.md#install-cloudify-manager" >}}) latest version on the host.
+  1. [Restore the snapshot]({{< relref "installation/upgrading-manager.md#restore-snapshot" >}}) of the Cloudify Manager to the host.
+  1. [Migrate agents]({{< relref "installation/upgrading-manager.md#migrate-agents" >}}) to the new Cloudify Manager.
 
 * Migration upgrade -
 
-  1. Save snapshot of the the Cloudify Manager.
-  1. Install the Cloudify Manager on the host.
-  1. Restore snapshot of the Cloudify Manager to the host.
-  1. (Optional) Uninstall (teardown in Cloudify 4.2 and below) the Cloudify Manager from the host.
-  1. Migrate agents from the old Cloudify Manager.
+  1. [Backup a snapshot]({{< relref "installation/upgrading-manager.md#backup-a-snapshot" >}}) of the Cloudify Manager.
+  1. [Backup the agents certificates]({{< relref "installation/upgrading-manager.md#backup-agent-certificates" >}}) from the Cloudify Manager.
+  1. [Install the Cloudify Manager]({{< relref "installation/upgrading-manager.md#install-cloudify-manager" >}}) latest version on the host.
+  1. [Restore the snapshot]({{< relref "installation/upgrading-manager.md#restore-snapshot" >}}) of the Cloudify Manager to the host.
+  1. (Optional) [Uninstall the Cloudify Manager]({{< relref "installation/upgrading-manager.md#uninstall-cloudify-manager" >}}) from the original host. In Cloudify 4.2 and below, this is called teardown.
+  1. [Migrate agents]({{< relref "installation/upgrading-manager.md#migrate-agents" >}}) to the new Cloudify Manager.
```

Comment:
After the agents install, the old manager should be powered off (whether or not it is deleted), and if auto-healing policies are used then it MUST be disconnected from all networks that could allow it to conduct healing workflows.

Reply:
I'll add a warning: If you do not uninstall the old Cloudify Manager and your workflows include auto-heal workflows, you must disconnect it from any networks that it can use to communicate with managed hosts.